apps/examples/slsiwifi/slsiwifi_main.c

```diff
@@ -104,7 +104,7 @@ static struct mm_heap_s *g_user_heap;
 int g_memstat_total = 0;
 static int getMemUsage(void)
 {
-	g_user_heap = mm_get_heap_info();
+	g_user_heap = mm_get_heap_info(NULL);
```

Comment:
NULL? Why NULL? Even if it is working with NULL, it looks strange. How about using new definition instead of NULL?